hexrd/ui/load_panel.py

```diff
@@ -77,8 +77,7 @@ def setup_gui(self):
         self.ui.file_options.resizeColumnsToContents()
 
     def setup_connections(self):
-        HexrdConfig().detectors_changed.connect(self.detectors_changed)
-        HexrdConfig().instrument_config_loaded.connect(self.config_changed)
```

Comment:
Since `LoadPanel.config_changed()` only does something if the detectors were changed, we are now hooking it up directly to `HexrdConfig().detectors_changed` instead, which should be emitted any time the detectors were changed.

 Also, since `LoadPanel.config_changed()` calls `LoadPanel.detectors_changed()`, only one connection was needed here.